### DISCUSSION TITLE
Call CLI commands in js code

### DISCUSSION THREAD

**Discussion by zfanta** [prisma/prisma#9028] [Q&A] [answered]
I am writing test codes with mysql docker image. Creating a new mysql docker container, I need to type `prisma migrate dev` in container.
I want to call migrate function in test setup code like `beforeAll` in Jest. Is there any other way than using `child_process.exec`?

**Comment by ryands17** [comment, marked as answer]
@zfanta 👋 

No other way apart from `exec` as of now. Watch this issue for more updates:
https://github.com/prisma/prisma/issues/4703

Also if you're running tests, you should use `prisma migrate deploy` to apply your migrations and not `prisma migrate dev`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
